ralph/afk.sh

```
@@ -0,0 +1,38 @@
+  exit 1
+fi
+
+# jq filter to extract streaming text from assistant messages
+stream_text='select(.type == "assistant").message.content[]? | select(.type == "text").text // empty | gsub("\n"; "\r\n") | . + "\r\n\n"'
+
+# jq filter to extract final result
+final_result='select(.type == "result").result // empty'
+
+for ((i=1; i<=$1; i++)); do
+  tmpfile=$(mktemp)
+  trap "rm -f $tmpfile" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)